CONTRIBUTING.rst

```diff
@@ -197,6 +218,7 @@ Once you are satisfied with your changes, you can run all the tests to make sure
 that your change did not break code elsewhere:
 
 .. code:: sh
+
```

Comment:
maybe another important pytest feature we could mention is running a specific test either with `pytest the_file.py::the_test` or `pytest the_file.py -k 'test_name_pattern'` to avoid waiting for all the other tests to run.


Also, I've noticed that in some cases in windows `pytest` can fail (I guess that depends on how it was installed) while `python -m pytest` seems a bit more robust. in the CI it is good to use `pytest` to avoid adding the current working directory to the python path but for local development I think `python -m pytest` is a good option; WDYT?

Reply:
also we could add a link to the pytest documentation

Reply:
added notes